src/index.html

```diff
@@ -9,5 +9,47 @@
   </head>
   <body>
     <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+    </div>
+    <div class="stars stars--2">
+      <div class="star__unit"></div>
```

Comment:
```suggestion
    <div class="stars stars--2">
      <div class="stars__unit"></div>
```
Это элемент внутри блока stars, соответственно сначала название блока, потом название элемента